tests/dialogBoxes.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from '@playwright/test';
+//1. Select the PET TYPES menu item in the navigation bar
+test.beforeEach(async ({ page }) => {
+    await page.goto('/')
+    await page.getByText('Pet Types').click()
+
+    //2. On the "Pet Types" page, add assertion of the "Pet Types" text displayed above the table with the list of pet types
+    await expect(page.getByRole("heading")).toHaveText("Pet Types")
+})
+
+test('add and delete pet type', async ({ page }) => {
+    //3. Click on "Add" button
+    await page.getByRole('button', { name: 'Add' }).click()
+
+    // 4. Add assertions of "New Pet Type" section title, 
+    await expect(page.locator('h2').nth(1)).toHaveText('New Pet Type')
+
+    //"Name" header for the input field
+    await expect(page.locator('label')).toHaveText('Name')
+
+    // the input field is visible
+    const inputValue = page.locator('#name')
+    await expect(inputValue).toBeVisible()
+
+    //5. Add a new pet type with the name "pig" and click "Save" button
+    await inputValue.click()
+    await inputValue.fill('pig')
```

Comment:
In most of the cases you don't need to `click()` into the input field before `fill()`
If you delete `click()` it will work the same way. `fill()` method is smart enough to handle it

Reply:
I love to make my code complex )))),  I thought I need to click first and then fill up the field